core/src/test/java/io/jenkins/pluginhealth/scoring/scores/SecurityWarningScoringTest.java

```diff
@@ -79,9 +132,11 @@ void shouldBeAbleToDetectPluginWithNoSecurityWarning() {
         final Plugin plugin = mock(Plugin.class);
         final SecurityWarningScoring scoring = getSpy();
 
-        when(plugin.getDetails()).thenReturn(Map.of(
-            KnownSecurityVulnerabilityProbe.KEY, ProbeResult.success(KnownSecurityVulnerabilityProbe.KEY, "No known security vulnerabilities.", 1)
-        ));
+        when(plugin.getDetails())
+                .thenReturn(Map.of(
+                        KnownSecurityVulnerabilityProbe.KEY,
+                        ProbeResult.success(
+                                KnownSecurityVulnerabilityProbe.KEY, "No known security vulnerabilities.", 2)));
```

Comment:
FYI the version of the probe is never asserted on scoring because we don't want to hard link minor probe changes to scoring validation.

Reply:
Restore @alecharp